ColumnMode/ColumnMode.vcxproj

```diff
@@ -23,33 +23,33 @@
     <ProjectGuid>{6546B3A0-08BA-43DD-A450-B2DE1262820A}</ProjectGuid>
     <Keyword>Win32Proj</Keyword>
     <RootNamespace>ColumnMode</RootNamespace>
-    <WindowsTargetPlatformVersion>10.0.17134.0</WindowsTargetPlatformVersion>

```

Comment:
Hate to do this, but can you remove this commit please? I'm not yet using VS2022.

Reply:
Dropped the commit :)